solr/contrib/ltr/src/java/org/apache/solr/ltr/ModelQuery.java

```diff
@@ -465,6 +466,27 @@ public ModelScorer scorer(LeafReaderContext context) throws IOException {
 
     }
 
+    public FeatureInfo[] somethingFeaturesInfo(int docid,
+        Float originalDocScore,
+        List<LeafReaderContext> leafContexts)
+        throws IOException {
+      final int n = ReaderUtil.subIndex(docid, leafContexts);
+      final LeafReaderContext atomicContext = leafContexts.get(n);
```

Comment:
The rescorer does something similar in the explain and rescore functions.  Mayeb this should be there instead?